### ISSUE TITLE
issue on iOS 9

### ISSUE THREAD

**Issue by phamhoang2812** [remirobert/TextDrawer#3]
i have a issue when build on real device (iOS 9 - iPhone 6) , it still work on simulator : 

Library not loaded: @rpath/Masonry.framework/Masonry
  Referenced from: /var/mobile/Containers/Bundle/Application/D31D044C-C5FF-4F25-A687-EF7278775175/Example.app/Example
  Reason: no suitable image found.  Did find: /private/var/mobile/Containers/Bundle/Application/D31D044C-C5FF-4F25-A687-EF7278775175/Example.app/Frameworks/Masonry.framework/Masonry: mmap() errno=1 validating first page of '/private/var/mobile/Containers/Bundle/Application/D31D044C-C5FF-4F25-A687-EF7278775175/Example.app/Frameworks/Masonry.framework/Masonry'

**Comment by remirobert** [comment]
Is it with the example project ?
😁

**Comment by phamhoang2812** [comment]
Yes , i downloaded your sample and then build on real device but it only stop at splash screen and show bug above .
